db/models/bundles.model.js

```diff
@@ -0,0 +1,33 @@
+const Sequelize = require('sequelize');
+const DataTypes = require('sequelize');
+const db = require ('../index.js');
+
```

Comment:
Same destructuring comment as your 'db/index.js' file, otherwise your Bundle const looks great.

Reply:
Point is well taken. This change will be made.